src/uu/stat/src/stat.rs

```diff
@@ -144,24 +175,39 @@ trait ScanUtil {
 }
 
 impl ScanUtil for str {
+    /// Scans for a number at the beginning of the string
+    /// Returns the parsed number and the character count (not byte count)
+    /// This was updated to properly handle UTF-8 by tracking both character and byte counts
```

Comment:
I thought a bit more about this change and somehow it doesn't make much sense to have both character and byte counts. We only deal with ASCII characters (`+`, `-`, and `0` - `9`) and thus character count and byte count are the same.

Reply:
yeah, good point!